frontend/packages/helm-plugin/src/components/list-page/HelmChartRepositoryHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import { sortable } from '@patternfly/react-table';
+import { TFunction } from 'i18next';
+import { Kebab } from '@console/internal/components/utils';
+
+const HelmChartRepositoryHeader = (t: TFunction) => () => {
+  return [
+    {
+      title: t('helm-plugin~Name'),
+      sortField: 'metadata.name',
+      transforms: [sortable],
+    },
+    {
+      title: t('helm-plugin~Display Name'),
+      sortField: 'spec.name',
+      transforms: [sortable],
+    },
+    {
+      title: t('helm-plugin~Namespace'),
+      sortField: 'metadata.namespace',
+      transforms: [sortable],
+    },
+    {
+      title: t('helm-plugin~Disabled'),
+      sortField: 'spec.disabled',
+    },
+    {
+      title: t('helm-plugin~Repo URL'),
+      sortField: 'spec.connectionConfig.url',
```

Comment:
Any reason to not add the `transforms: [sortable]` to `Disabled` and `Repo URL` columns?

Reply:
Updated the PR